waiter/src/waiter/scheduler.clj

```diff
@@ -47,6 +47,11 @@
   [& args]
   `(log/log "Scheduler" :debug nil (print-str ~@args)))
 
+(defmacro log-service-instance
+  "Log InstanceTracker-specific messages."
+  [instance event-type]
+  `(log/log "InstanceTracker" :debug nil (utils/clj->json (assoc ~instance :timestamp (t/now) :event-type ~event-type))))
```

Comment:
Why is this set at `debug` by default?

Reply:
To preserve the convention that Scheduler specific messages use debug by default. 

```
(defmacro log
  "Log Scheduler-specific messages."
  [& args]
  `(log/log "Scheduler" :debug nil (print-str ~@args)))
```

Reply:
OK, I like the idea of setting all of our current messages to `:info` level, and reserving the ability to set some more noisy events later to `:debug` level. @pandashadopan , let's met the log level an explicit argument to this macro.

I also with Shams that we should `select-keys` on the instance to choose the fields we want to log.